guidance/models/_model.py

```diff
@@ -1323,3 +1323,11 @@ def _check_dominated(node, parser, match_version, next_byte_mask):
             if not child_dominate:
                 return False
     return True
+
+
+class RemoteModel(Model):
+    def __init__(self, endpoint, echo=True, **kwargs):
+        from ._remote import RemoteEngine
+
+        engine = RemoteEngine(endpoint, **kwargs)
+        super().__init__(engine, echo=echo)
```

Comment:
@riedgar-ms we're thinking we can just have users pass in a URL to the Model subclass directly to indicate a RemoteModel, instead of needing another layer

Reply:
So users would just do
```python
m_remote = models.Model("https://my.url.endpoint/")
```
?

Reply:
Yes (with appropriate auth params if needed too)

Reply:
I don't think users will want to use models.Model directly, because the client needs to know what kind of syntax to use for the model etc. So while we allow the base class to contain this, note that it mostly used in the subclass constructors

Reply:
I would have put `RemoteModel` back if this hadn't already been merged.....